### ISSUE TITLE
Neither the credentials or download process initialize

### ISSUE THREAD

**Issue by afcotroneo** [hdus/SRTM-Downloader#12]
Neither the credentials or download process initialize when a user selects download from the prompt window

**Comment by afcotroneo** [comment]
This is consistent across multiple versions of qgis and different OS'

**Comment by Elstermann** [comment]
same problem here
Win7 64-bit | QGIS 3.4.15

**Comment by hdus** [comment]
At the moment I'm not able to fix this issue and any other issues due to I'm on a sabbatical around the world. I would appreciate any posted fixes or pull requests.

**Comment by talledodiego** [comment]
The problem is that the following url is not valid:
url = "https://e4ftl01.cr.usgs.gov//MODV6_Dal_D/SRTM/SRTMGL1.003/2000.02.11/%s%s.SRTMGL1.hgt.zip" % (lat_tx, lon_tx)
I was not able to find a new valid url to those files, because I generally download those files from EarthExplorer and there there are no hgt files.

**Comment by hdus** [comment]
Are you able to open this URL?

https://e4ftl01.cr.usgs.gov//MODV6_Dal_D/SRTM/SRTMGL1.003/2000.02.11/N46E007.SRTMGL1.hgt.zip

**Comment by talledodiego** [comment]
no, 
"impossible to find IP address of the server urs.earthdata.nasa.gov".

**Comment by zelincu** [comment]
> 
> 
> Are you able to open this URL?
> 
> https://e4ftl01.cr.usgs.gov//MODV6_Dal_D/SRTM/SRTMGL1.003/2000.02.11/N46E007.SRTMGL1.hgt.zip

Yes.
But i have same issue here, cant download images or insert credentials.

**Comment by talledodiego** [comment]
> > Are you able to open this URL?
> > https://e4ftl01.cr.usgs.gov//MODV6_Dal_D/SRTM/SRTMGL1.003/2000.02.11/N46E007.SRTMGL1.hgt.zip
> 
> Yes.
> But i have same issue here, cant download images or insert credentials.

Are you sure you can open it? In several PCs I've tried it says always "impossible to find IP address of the server ..."

**Comment by zelincu** [comment]
> 
> 
> > > Are you able to open this URL?
> > > https://e4ftl01.cr.usgs.gov//MODV6_Dal_D/SRTM/SRTMGL1.003/2000.02.11/N46E007.SRTMGL1.hgt.zip
> > 
> > 
> > Yes.
> > But i have same issue here, cant download images or insert credentials.
> 
> Are you sure you can open it? In several PCs I've tried it says always "impossible to find IP address of the server ..."

Just downloaded the zip file again

**Comment by hdus** [comment]
When you open the URL https://e4ftl01.cr.usgs.gov//MODV6_Dal_D/SRTM/SRTMGL1.003/2000.02.11/N46E007.SRTMGL1.hgt.zip in your internet browser you should be directed to a login window. If not something with your network is misconfigured probabely.

**Comment by drbiosis** [comment]
Was anybody successful on solving this issue?

**Comment by drbiosis** [comment]
> Was anybody successful on solving this issue?

UPDATE: I tried openning the URL and didn't work at first. I gave permission since it was blocked by default and then everything worked fine.

**Comment by hdus** [comment]
Sometimes it takes a long time to open the credentials window. Please be patient. It will work!

**Comment by temospena** [comment]
I'm having the same issue...
It does not open a popup window with credentials, even if I wait for 1h, even for different computers with different QGIS versions.

But it was working properly two days ago.
Thanks for the plugin!

**Comment by temospena** [comment]
I can access the file that I want through https://search.earthdata.nasa.gov/ although I cannot download it even with an available url for my file.
So, maybe this is the issue...
![image](https://user-images.githubusercontent.com/39107166/97187324-3f56fd00-179a-11eb-8fa8-be8754b5b7a6.png)

**Comment by JasonDalton** [comment]
I think I'm seeing the same behavior on Mac with QGIS 3.16 and SRTM Downloader v3.1.9.
I enter all the form parameters, click Download, and the credentials dialog never appears.   The plugin seems to check each image needed, gets denied, and then says it's done, but with no tiles listed in the plugin window, no data loaded in QGIS, and no data downloaded to the save path.

**Comment by Manos1981** [comment]
I see a different issue. Qgis 3.16.3. Till last week it ws working normally... Now it doesnt show the credentials screen therefore the download progress bar fills up but no images are downloaded at all! I checked maybe was something with my firewall but no qgis at least is free to do whatever through it... any ideas?

**Comment by jendahorak** [comment]
Same problem here... on multiple computers (linux/windows). Completion bar fills up but no images are downloaded.

**Comment by ngedigk** [comment]
The same happens here no matter what version QGIS 3.16.0. or QGIS 3.16.3.

**Comment by Mikester-Calgary** [comment]
Confirmed!! Can we get one of the mods to please reopen this thread?  (If I don't see anything open tomorrow I will open a new Issue.)
It appears that something has changed on NASAs side.  The server at NASA is now up again but still no service to the plugin.

**Comment by hdus** [comment]
NASA has changed the download URL. It's fixed with release 3.1.10.

**Comment by myPig0113** [comment]
> NASA has changed the download URL. It's fixed with release 3.1.10.

Why can't I find version 3.1.10

**Comment by hdus** [comment]
It's published at the QGIS Plugin Repository and it will be available soon there. It takes some time between upload and availability.

**Comment by myPig0113** [comment]
> It's published at the QGIS Plugin Repository and it will be available soon there. It takes some time between upload and availability.


Thank you. I'm looking forward to the new version

**Comment by Mikester-Calgary** [comment]
Thank you sir.  You’re the best!

M-C

**Comment by Manos1981** [comment]
Yes all good now its working like a charm!

**Comment by JasonDalton** [comment]
When does the update come out for the Mac?

**Comment by hdus** [comment]
Please use the QGIS Plugin Manager for upgrading the plugin.

**Comment by gdumouchel** [comment]
I appreciate the hard work going on here but the issue (perhaps different issue? I can open a new one if desired) is unfortunately still not resolved for me. After entering credentials (which successfully log on to the USGS website), the dialogue box immediately reappears. Attempting to close the dialogue box with "cancel" or using the "X" in the top right corner it will continuously reappear many times, sometimes forcing me to force close the program.

The behavior is the same in the LTS 3.10.14 as well as the current 3.16.3, both using the most current version 3.1.10 of the plugin on a Win10 machine.

**Comment by lifanchu** [comment]
I am having a similar issue in 3.16.4. There is no download process and no credentials window when I click "Download".
